main.js

```diff
@@ -59,7 +76,58 @@ function getCurrentGutter() {
 	return getGridProperties().gutter;
 }
 
+/**
+ * This sets MediaQueryListeners on all the o-grid breakpoints
+ * and fires a `o-grid.layoutChange` event on layout change.
+ */
+function enableLayoutChangeEvents() {
+	// Create a map containing all breakpoints exposed via html:before
+	const gridLayouts = getGridBreakpoints();
+	if (gridLayouts.hasOwnProperty('layouts')) {
+		const layouts = gridLayouts.layouts;
+		const breakpoints = new Map(Object.keys(layouts).map(key => [key, layouts[key]]));
```

Comment:
@JakeChampion That reminds me, don't merge this until the issue with Map and Sort on the polyfill service is worked out! I think it's currently broken in IE, not sure if there's a resolution for that yet — had to revert a bunch of similar code to Arrays the other day while working financial-times/next-frontpage#1198. Will check tomorrow and update.